pages/components/modal.html

```diff
@@ -106,8 +106,67 @@ <h1 clss="page-title">Modal</h1>
 
 
   <main class="main-content">
-
-    <iframe src="/codeblock/x.txt" frameborder="0" class="codeblock"></iframe>
+    <aside class="modal">
+      <ul class="list list-stacked modal-content">
```

Comment:
this works fine, but you can use a simple div and then use display : flex for the same purpose, as when it would be easy to implement the component like that

Reply:
I wanted to use semantic